awx/ui/package.json

```diff
@@ -29,7 +29,7 @@
     "react-error-boundary": "^3.1.3",
     "react-router-dom": "^5.1.2",
     "react-virtualized": "^9.21.1",
-    "rrule": "^2.6.4",
+    "rrule": "2.6.4",
```

Comment:
Pinning `rrule` due to a known bug with v2.6.6 and higher: https://github.com/jakubroztocil/rrule/issues/427